packages/state/src/index.ts

```diff
@@ -947,6 +958,16 @@ export function prepareItems(context: GroupMachineContextValue): Item[] {
       continue;
     }
 
+    if (item.lastKnownSize) {
+      const lastSize = makePixelUnit(
+        context.orientation === "horizontal"
+          ? item.lastKnownSize.width
+          : item.lastKnownSize.height
+      );
+      newItems.push({ ...item, currentValue: lastSize });
+      continue;
+    }
```

Comment:
When we prepare for updating the layout we use the last updated size if it exists.